Requesting user-defined output variables (#1003)
NREL/OpenStudio-HPXML: docs/source/workflow_outputs.rst

   Zone Temperatures                    Average temperatures (in deg-F) for each space modeled (e.g., living space, attic, garage, basement, crawlspace, etc.).
   Airflows                             Airflow rates (in cfm) for infiltration, mechanical ventilation (including clothes dryer exhaust), natural ventilation, whole house fans.
   Weather                              Weather file data including outdoor temperatures, relative humidity, wind speed, and solar.

>    Output Variables                     These are optional and user-defined. For advanced variables, the ``--add-component-loads`` argument must be used.

Reviewer: I think it's pretty unusual to request advanced EnergyPlus output variables and think this language will just confuse most users. If a user is requesting an advanced variable, they should know what they're doing.

```suggestion
   EnergyPlus Output Variables          These are optional and can be requested with the ReportSimulationOutput ``user_output_variables`` argument.
```
